pymc_marketing/clv/models/pareto_nbd.py

```diff
@@ -166,44 +166,21 @@ def __init__(
         model_config: Optional[Dict] = None,
         sampler_config: Optional[Dict] = None,
     ):
-        # Assign inputs to attributes and perform validation checks
-        try:
-            self.customer_id = data["customer_id"]
-            if len(np.unique(self.customer_id)) != len(self.customer_id):
-                raise ValueError("Customers must have unique ID labels.")
-            self.coords = {"customer_id": self.customer_id}
-        except KeyError:
-            raise KeyError("customer_id column is missing from data")
-        try:
-            self.frequency = data["frequency"]
-        except KeyError:
-            raise KeyError("frequency column is missing from data")
-        try:
-            self.recency = data["recency"]
-        except KeyError:
-            raise KeyError("recency column is missing from data")
-        try:
-            self.T = data["T"]
-        except KeyError:
-            raise KeyError("T column is missing from data")
-
+        warnings.warn(
+            "The Pareto/NBD model is still experimental. Please see code examples in documentation if model fitting issues are encountered.",
+            UserWarning,
+        )
```

Comment:
Why is this warning being re-added? For https://github.com/pymc-labs/pymc-marketing/pull/545?

Reply:
Probably a git conflict mistake on my part. I started working on this PR on a super stale main branch xD. Will revert

Reply:
Fixed